nx-X11/lib/X11/Imakefile

```diff
@@ -891,6 +891,13 @@ OBJS3 = \
 	xim_trans.o \
 	$(XKBOBJS) $(LCOBJS)
 
+SHAREDLCDEPSYMSRCS = XlcDL.c XlcSL.c
+
+SHAREDI18NSTATICSRCS = \
+        XDefaultIMIF.c XDefaultOMIF.c
+
+STATICLCDEPSYMSRCS =
```

Comment:
@uli42: Why do we want to re-introduce 'XDefaultIMIF.c' and 'XDefaultOMIF.c'??? Not getting the point here. Please comment and improve the commit message, if you are really intending to revert this.